docs/UsersGuide.asciidoc

```diff
@@ -217,6 +217,12 @@ builds but rather deprioritize them up to a configurable limit of priority.
 _DEPRIORITIZE_LIMIT:: The configurable limit of priority up to which jobs
 should be deprioritized. Needs `_DEPRIORITIZEBUILD`. Default 100.
 
+_ONLYOBSOLETESAME:: Only obsolete (or deprioritize) jobs for the same BUILD.
```

Comment:
I struggle with the word "SAME". Can we find better names than just "BUILD" and "SAME"? I guess the name "_NOOBSOLETEBUILD" should be adjusted accordingly already. I would be ok to change the old variable as well because it is kind of an "internal" one. Do you have good suggestions for the both variables?

Reply:
IAGREETHATTHEWORD is hard to parse

Reply:
I agree they're not the best names, but I can't immediately think of anything better :/ gimme half an hour and I'll let it percolate a bit.

Reply:
Random idea for names:`_NO_JOB_BUILD_OBSOLENCE=1` and `_EQUAL_JOB_BUILD_OBSOLENCE=1`?

Reply:
It's OBSOLESCENCE . But yeah, not bad...

Reply:
Potato, potato :) :P

Reply:
hm, I opt for just "_NO_OBSOLETE" and "_ONLY_OBSOLETE_SAME_BUILD".

1. OBSOLESCENSE is hard to type right apparently :)
2. underscore_separation_helps_reading
3. I would not mention "job" when it's most likely about a whole lot of them, as in "a whole build"
4. "no obsolete build" was ambiguous already because not a "build" was obsoleted but any "build" with matching other product parameters